openroad_interface/macro_maker.py

```diff
@@ -0,0 +1,148 @@
+import math
+import copy
+import yaml
+
+from functions import *
+
+lef_tech_file = "OpenROAD/test/Nangate45/Nangate45_tech.lef" 
```

Comment:
this should not be hardcoded. 
We want this script to work for any specified lef file